### ISSUE TITLE
Update spartacus-installation to angular 9

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6714]
Update spartacus-installation to use angular 9. 

Make sure it works on the server. 

---
For server deployment:
spartacus-installation hasn't been updated to use angular 9.
So I manually updated the package.json to use angular 9.
So now it deploys without errors, but I still get these warnings, and the storefront does not load. I can't remove those files, because at least environment.prod.ts is needed.
Anyone has any idea???
 ```
WARNING in /data/hybris/spartacus-installation/app/spartacus/src/app/app-routing.module.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.
WARNING in /data/hybris/spartacus-installation/app/spartacus/src/environments/environment.prod.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.
```

**Comment by marlass** [comment]
Hey, I prapared some fixes on branch `2.0` in spartacus-installation. You could change branch for the script on server and then test develop deployment.

Warning: it is only prepared for static deployment (no ssr)

**Comment by KateChuen** [comment]
it works! No more warnings.
